mock/py/mockbuild/backend.py

```diff
@@ -684,9 +684,10 @@ def _rpmbuild_noclean_option(self):
             return self.rpmbuild_noclean_option
 
         self.rpmbuild_noclean_option = ""
-        out, status = self.buildroot.doChroot(["rpmbuild", "--help"],
-                                              returnOutput=True)
-        if not status and "--noclean" in out:
+        out, _ = self.buildroot.doChroot("rpmbuild --help | grep -- --noclean",
```

Comment:
why the grep? Isnt starting another process more expensive than parsing multiline output?